gcloud-java-dns/src/main/java/com/google/gcloud/dns/spi/DefaultDnsRpc.java

```diff
@@ -65,7 +65,7 @@ private DefaultRpcBatch(BatchRequest batch) {
     }
 
     @Override
-    public void addListZones(DnsRpc.Callback<ManagedZonesListResponse> callback,
+    public void addListZones(RpcBatch.Callback<ManagedZonesListResponse> callback,
```

Comment:
Nit: you could consider importing `Callback`, that should make the code in this class slightly more compact